feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/input/ui/audiorecorder/AudioRecorderViewModel.kt

```diff
@@ -0,0 +1,176 @@
+package com.hedvig.android.odyssey.input.ui.audiorecorder
+
+import android.media.MediaPlayer
+import android.media.MediaRecorder
+import androidx.lifecycle.ViewModel
+import java.io.File
+import java.time.Clock
+import java.time.Instant
+import java.util.*
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+
+class AudioRecorderViewModel(
```

Comment:
Aha I see. This class exists in this code-base, but it actually only exists inside the :app module for now, making it unreachable from here, so we have to have this in two places until we extract this to a place where both of these modules have access to it.

Can be smth we should pickup in the future too 📝